README.md

```diff
@@ -78,7 +79,7 @@ After resolving those you should be able to run `make run_samples_without_docker
 ```pwsh
 # Indicate path to your FFMPEG installation (with subfolders `bin` with DLLs, `include`, `lib`)
 $env:SKBUILD_CONFIGURE_OPTIONS="-DTC_FFMPEG_ROOT=C:/path/to/your/ffmpeg/installation/ffmpeg/" 
-
+pip install scikit-build
```

Comment:
should not be necessary

Reply:
if older versions of setuptools can't do this, then it probably better to error out